### PR TITLE
Sofia/enums

### PR DESCRIPTION
**Pull request by sofiiako**

### CODE REVIEW

File: Enums/Enums and Pattern Matching/Intro/task.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Enums and Pattern Matching
```
Comment (Geravant) [inline Enums/Enums and Pattern Matching/Intro/task.md:1]: The task-info.yaml is missing for this task

File: Enums/Enums and Pattern Matching/Implement Match/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,58 @@
+pub enum Message {
```
Comment (Geravant) [inline Enums/Enums and Pattern Matching/Implement Match/src/lib.rs:1]: Should we add a main to demonstrate the results in this task?

**Review by Geravant — CHANGES_REQUESTED** [review]

**Comment by sofiiako** [comment]
@Geravant Addressed the issues, see the commits above :)

**Review by Geravant — APPROVED** [review]

File: Enums/Enums and Pattern Matching/The Match Operator/task.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+```
+
+The `Some(5)` value doesn’t match the pattern `None`, so we continue to the
+next arm.
+
+```rust,ignore
+Some(i) => Some(i + 1),
+```
+
+Does `Some(5)` match `Some(i)`? Why yes it does! We have the same variant. The
```
Comment (stephen-hero) [inline Enums/Enums and Pattern Matching/The Match Operator/task.md:194]: Why, yes it does!

File: Enums/Enums and Pattern Matching/The Match Operator/task.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+match some_u8_value {
+    1 => println!("one"),
+    3 => println!("three"),
+    5 => println!("five"),
+    7 => println!("seven"),
+    _ => (),
+}
+```
+
+The `_` pattern will match any value. By putting it after our other arms, the
```
Comment (stephen-hero) [inline Enums/Enums and Pattern Matching/The Match Operator/task.md:270]: When put after our other arms, the

**Pull request merged**